src/pages/[platform]/build-a-backend/existing-resources/cdk/index.mdx

```diff
@@ -0,0 +1,865 @@
+                'swift',
+                'vue'
+              ]
+};
+
+export const getStaticPaths = async () => {
+  return getCustomStaticPath(meta.platforms);
+};
+
+export function getStaticProps(context) {
+  return {
+    props: {
+      platform: context.params.platform,
+      meta
+    }
+  };
+}
+
+This guide shows you how to connect new mobile and web apps to AWS resources you've already created using the [AWS Cloud Development Kit (AWS CDK)](https://aws.amazon.com/cdk/). The AWS CDK is an open-source software development framework for defining cloud infrastructure as code with modern programming languages. This infrastructure is then deployed through [AWS CloudFormation](https://aws.amazon.com/cloudformation/). 
+
+The guide has three sections to help you build and connect a fullstack app:
+
+1. [Section one](/[platform]/build-a-backend/existing-resources/cdk/#build-a-graphql-api-using-the-amplify-data-cdk-construct) provides instructions for using the Amplify Data CDK to create a GraphQL API backend with AWS AppSync. This creates the core backend.
+1. [Section two](/[platform]/build-a-backend/existing-resources/cdk/#build-a-flutter-app-and-connect-to-the-graphql-api) shows how to build and integrate a Flutter app with the GraphQL API. Follow this if you are using Flutter to build a cross-platform frontend.
+1. [Section three](/[platform]/build-a-backend/existing-resources/cdk/#build-a-react-app-and-connect-to-the-graphql-api) explains how to create and connect a React web app to the GraphQL API. Follow this if you are using React to build a web frontend.
+
+The recommended approach is to complete section one to build the backend, then optionally complete section two (Flutter) or three (React), or both, to connect frontends to the GraphQL API. Once you're finished experimenting with these demo apps, we recommend visiting the ["Clean up resources"](/[platform]/build-a-backend/existing-resources/cdk/#clean-up-resources) section to delete the backend resources to avoid incurring unexpected costs. 
+
+Before you begin, you will need:
+
+* An AWS account: If you don't already have an account, follow the [Setting Up Your AWS Environment](https://aws.amazon.com/getting-started/guides/setup-environment/) tutorial for a quick overview.
+* The Amplify CLI [installed](https://docs.amplify.aws/cli/start/install/) and configured.
+* A text editor. For this guide, we will use VS Code, but you can use your preferred IDE.
+* Flutter and its command-line tools [installed](https://docs.flutter.dev/get-started/install) and configured (if completing the second part of this guide).
+
+## Build a GraphQL API using the Amplify Data CDK construct
+
+The CDK provides a simple way to define cloud infrastructure in code. In this section, we will use the CDK to build out the backend resources for our application.
+
+**Step 1:** Create a folder for the CDK app by running the following command in your terminal. 
+
+```bash showLineNumbers={false}
+
```

Comment:
these newlines don't appear to affect the the display of the code snippet on the site, but for consistency with other pages can we remove the surrounding newlines in code snippets?